scripts/travis/es-integration-test.sh

```diff
@@ -22,7 +22,10 @@ make build-crossdock-ui-placeholder
 GOOS=linux make build-query
 
 make test-compile-es-scripts
+CID=$(docker run --rm -d -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" -e "xpack.security.enabled=false" -e "xpack.monitoring.enabled=false" docker.elastic.co/elasticsearch/elasticsearch:7.3.0)
```

Comment:
Why is this needed?

Reply:
I was just playing around with things and locally this was required (no ES was running / mocked on port 9200 in order for those test to run against)

Honestly I am a little lost  ... I simply tried the same approach that was done a few lines above in that script in order to make things work.